pkg/tags/tag_test.go

```diff
@@ -189,30 +189,40 @@ func TestTagsMultipleConcurrentIncrementsSyncMap(t *testing.T) {
 	n := 100
 	wg := sync.WaitGroup{}
 	wg.Add(10 * 5 * n)
+	errC := make(chan error)
```

Comment:
both of these channels are not necessary. you can just call `t.Errorf(..)` on line 206, then just wait on the waitgroup outside the for loop. the extra channels are not needed

Reply:
t.Fatal was complaining... t.Errorf is not... Fixed. Thanks